quiz.css

```diff
@@ -0,0 +1,66 @@
+html {
+	margin: 0;
+	padding: 0;
+}
+
+.productCard {
+	border: 1px solid lightblue;
+	width: 30%;
+	height: 500px;
```

Comment:
This height is actually hurting you a little. If you view it without the dev console open or on a large screen, your text pokes out the bottom.